tests/integration/workflows/test_task_matching.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import os
+import json
+import pytest
+
+from invenio_search import current_search_client as es
+from invenio_workflows import workflow_object_class
+from inspirehep.modules.migrator.tasks import record_insert_or_replace
+from inspirehep.modules.workflows.tasks.matching import article_exists
+
+
+@pytest.fixture(scope="function")
+def setup_record(workflow_app):
+    parent_folder = os.path.abspath(os.path.join(__file__, os.pardir))
+    with open(os.path.join(parent_folder, 'fixtures', 'matcher_rec.json')) as f:
+        rec = record_insert_or_replace(json.loads(f.read()))
+        es.indices.refresh('records-hep')
+        yield rec
+        rec._delete(force=True)
+
+
+def _create_hep_wf(to_match):
+    return workflow_object_class.create(
+        data_type='hep',
+        id_user=1,
+        data=to_match
+    )
+
+
+def test_match_with_invenio_matcher_matches_a_record(setup_record):
```

Comment:
This probably should be named `test_article_exists_...`, as it's actually testing the article_exists function.